script/build.js

```diff
@@ -451,7 +451,7 @@ if (options.watch) {
         { from: '^/health-care/health-records(.*)', to: '/health-care/health-records/' },
         { from: '^/health-care/messaging(.*)', to: '/health-care/messaging/' },
         { from: '^/health-care/prescriptions(.*)', to: '/health-care/prescriptions/' },
-        { from: '^/letters(.*)', to: '/letters/' },
+        { from: '^/download-va-letters(.*)', to: '/download-va-letters/letters/' },
```

Comment:
We will want to add `/letters` to the `from`, I think.
{ from: '^/download-va-letters/letters(.*)', to: '/download-va-letters/letters/' }

Reply:
Actually this causes `Cannot GET` errors.